### PR TITLE
Typos / Grammar French translation

### PR DESCRIPTION
**Pull request by gabparrot**
Fixed several typos, bad accents, etc. Very low priority

### CODE REVIEW

**Comment by gabparrot** [comment]
There is still the **"Backup Secret Phrase"** on the main screen menu that appears in plain English, but I didn't know which variable it is. French would be "Sauvegarder la phrase secrète". If it's too long for the field, perhaps "Noter la phrase secrète" could be a shorter alternative.

**Comment by bbedward** [comment]
Feel free to send us an email at hello@appditto.com if you'd like to continue with these revisions, we manage everything on poeditor.com now
